generated/java/proto-google-appengine/build.gradle

```diff
@@ -0,0 +1,168 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'signing'
+
+description = 'PROTO library for proto-google-appengine-v1'
+group = "com.google.api.grpc"
+version = "0.1.15"
```

Comment:
This should be 0.1.18. The required updates in googleapis to ensure 0.1.18 is generated are in this PR: https://github.com/googleapis/googleapis/pull/455 but the googleapis changes need to be made upstream - there is an internal change pending.